WooCommerce/Classes/POS/Presentation/POSFloatingControlView.swift

```diff
@@ -110,5 +122,26 @@ private extension POSFloatingControlView {
             value: "Get Support",
             comment: "The title of the floating button to get support for Point of Sale, shown in a popover menu."
         )
+
+        static let viewDocumentation = NSLocalizedString(
+            "pointOfSale.floatingButtons.viewDocumentation.button.title",
+            value: "View Documentation",
```

Comment:
We're changing this string to "Documentation" only. Ref: p1738915805063569?thread_ts=1738652142.617639&cid=C070SJRA8DP-slack-C070SJRA8DP I'll make the change before merging, in case I have to address something else 👍